### ISSUE TITLE
Stop running simulation for design days

### ISSUE THREAD

**Issue by asparke2** [NatLabRockies/ComStock#75]
Currently, ComStock runs the simulations for design days. This adds additional timeseries outputs which must be removed during postprocessing. It also presumably takes additional computation time, although how much is unclear. Not sure if this change needs to happen in ComStock simulation settings or in openstudio-standards.

**Comment by brianlball** [comment]
@asparke2 have you all worked on this issue at all?

**Comment by asparke2** [comment]
No, but I think the solution is as simple as changing the field `Run Simulation for Sizing Periods` in the `SimulationControl` object from "Yes" to "No"
